modules/gltools/src/context/context.js

```diff
@@ -176,7 +176,8 @@ function createBrowserContext(canvas, options) {
   const {onError} = options;
 
   // Try to extract any extra information about why context creation failed
-  const onCreateError = error => onError(`WebGL context: ${error.statusMessage || 'error'}`);
+  let errorMessage = null;
+  const onCreateError = error => (errorMessage = error.statusMessage || errorMessage);
```

Comment:
Is `error.statusMessage`  populated by webgl create context error callback? Might be worth mentioning where this comes from...

Reply:
Yes https://developer.mozilla.org/en-US/docs/Web/API/HTMLCanvasElement/webglcontextcreationerror_event#:~:text=The%20webglcontextcreationerror%20event%20of%20the,more%20information%20about%20the%20failure.